src/components/TrackCardStack.test.tsx

```
@@ -0,0 +1,248 @@
+        queryClient = new QueryClient({
+            defaultOptions: {
+                queries: {
+                    retry: false,
+                },
+            },
+        });
+        mockFetch.mockReset();
+        mockPushToast.mockReset();
+
+        // Reset audio player mocks
+        Object.values(mockAudioPlayer).forEach(mock => mock.mockReset());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)